index.html

```diff
@@ -66,74 +64,37 @@
-            create and manage microVMs. Firecracker has a minimalist design. It excludes unnecessary devices and guest
-            functionality to reduce the memory footprint and attack surface area of each microVM. This improves
-            security, decreases the startup time, and increases hardware utilization. Firecracker currently supports
-            Intel CPUs, with AMD and Arm support in developer preview.
+            Firecracker is a virtual machine monitor (VMM) that uses the Linux Kernel-based Virtual Machine (KVM) to create and manage microVMs. Firecracker has a minimalist design. It excludes unnecessary devices and guest functionality to reduce the memory footprint and attack surface area of each microVM. This improves security, decreases the startup time, and increases hardware utilization. Firecracker currently supports Intel CPUs, with AMD and Arm support in developer preview.
           </p>
           <p>
-            Firecracker is integrated with
-            <a
-              href="https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support"
-            >
-              Kata Containers
-            </a>
-            ,
-            <a href="https://opennebula.io/firecracker/">Open Nebula</a>
-            ,
-            <a href="https://github.com/solo-io/unik">UniK</a>
-            ,
-            <a href="https://www.weave.works/oss/firekube/">Weave FireKube</a>
-            (via
-            <a href="https://github.com/weaveworks/ignite">Weave Ignite</a>
-            ), and containerd via
-            <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>
-            It's also used by
-            <a href="https://fly.io/">fly.io</a>
-            . Firecracker can run Linux and
-            <a href="http://blog.osv.io/blog/2019/04/19/making-OSv-run-on-firecraker">OSv</a>
-            guests. Our latest roadmap can be found
-            <a href="https://github.com/firecracker-microvm/firecracker/projects/13">here</a>
-            .
+            Firecracker is integrated with <a href="https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support">Kata Containers</a>, <a href="https://opennebula.io/firecracker/">OpenNebula</a>, <a href="https://github.com/solo-io/unik">UniK</a>, <a href="https://www.weave.works/oss/firekube/">Weave FireKube</a> (via <a href="https://github.com/weaveworks/ignite">Weave Ignite</a>), and containerd via <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>. It is also being used by <a href="https://fly.io">Fly.io</a>. Firecracker can run Linux and <a href="http://blog.osv.io/blog/2019/04/19/making-OSv-run-on-firecraker">OSv</a> guests. Our latest roadmap can be found <a href="https://github.com/firecracker-microvm/firecracker/projects/13">here</a>.
```

Comment:
Based on the two previous comments, this paragraph and subsequent ones like it need both elements on their own lines, and text line breaking.